.travis.yml

```diff
@@ -14,6 +14,9 @@ install:
 
 script:
   - make verify BRANCH=$TRAVIS_BRANCH
+  - ./hack/test-go.sh
```

Comment:
This is going to take twice as long as Shippable, since it'll run all of the unit tests multiple times (once for each KUBE_TEST_API_VERSIONS * KUBE_TEST_ETCD_PREFIXES combination).

The last time we ran unit tests on Travis, we used a matrix to only run one of the combinations at a time, I think.